app/models/concerns/state.rb

```diff
@@ -15,4 +15,9 @@ module State
   CLOSED_STATE = 'closed'
   SELECTABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE, HIDDEN_STATE]
   SEARCHABLE_STATES = [OPEN_STATE, CLOSED_STATE, REJECTED_STATE]
+  TODO_LIST_STATES = [SPONSORED_STATE, VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS_STATES = [VALIDATED_STATE, PENDING_STATE]
+  COLLECTING_SPONSORS = 'collecting sponsors'
```

Comment:
I am not sure if it is a good idea to mix real states (meaning those persisted in the DB) with these filter keywords. Maybe there could be a better place for these constants? Or they could be named clearly to not be confused with real states. 

Mixing these in the controller and model is a bit confusing as well. Of course this is hard if we want to stick with the single `select` html element (for filtering with a single state and a group of  states).

Just some thoughts, I don't know if any action is needed. What do you think?